zkPySyft/core/zokrates.py

```diff
@@ -74,8 +99,19 @@ def run(self, path):
         self.compute_witness()
         self.generate_proof()
 
+    def __write_gadget_code_files(self):
+        gadgets = self.gadgets
+        for g in gadgets:
+            with open(g.name + ".code", "w+") as f:
+                f.write(g.dsl_code)
+
     def __write_code_file(self, path):
         with open(path, "w+") as f:
+            # add imports to header
```

Comment:
Minor: by convention comments should start with an uppercase (https://www.python.org/dev/peps/pep-0008/#comments)